lib/runtime/wasmer/imports_test.go

```diff
@@ -56,6 +57,17 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+func Test_ext_offchain_timestamp_version_1(t *testing.T) {
+	inst := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
+
+	ret, err := inst.Exec("ext_offchain_timestamp_version_1", nil)
+	require.NoError(t, err)
+
+	expected := time.Now().Unix()
+
+	require.GreaterOrEqual(t, expected, ret)
```

Comment:
this seems like it might fail occasionally if the time difference between calling the runtime function and then doing `time.Now()` here happens over two different seconds, maybe give this a buffer of 1s?

Reply:
Should it, though? Since the following code blocks happen sequentially (not in goroutine): 	
```
ret, err := inst.Exec("rtm_ext_offchain_timestamp_version_1", nil)
....
expected := time.Now().Unix()
```
"expected" should always have greater value than "ret" value.

Reply:
misread this, thought it was `require.Equal`. yep this should be good!